posthog-node/src/feature-flags.ts

```diff
@@ -112,6 +114,35 @@ class FeatureFlagsPoller {
     return response
   }
 
+  async getFeatureFlagPayload(
```

Comment:
for the interface, we'll need a distinct_id here as well, since node is stateless.

Not sure what `value` represents?

Reply:
ugh, nvm, this isn't the interface 🤦 . I'm going to stay logged off tonight and check this tomorrow 😂